graylog2-web-interface/src/views/pages/SavedSearchPage.jsx

```diff
@@ -0,0 +1,31 @@
+// @flow strict
+import * as React from 'react';
+
+import ViewLoaderContext from 'views/logic/ViewLoaderContext';
+import NewViewLoaderContext from 'views/logic/NewViewLoaderContext';
+import { ExtendedSearchPage } from 'views/pages';
+import { loadNewView as defaultLoadNewView, loadView as defaultLoadView } from 'views/logic/views/Actions';
+import IfUserHasAccessToAnyStream from 'views/components/IfUserHasAccessToAnyStream';
+
+type Props = {
+  loadNewView?: () => mixed,
+  loadView?: (string) => mixed,
+  route: any,
+};
+
+const SavedSearchPage = ({ loadNewView = defaultLoadNewView, loadView = defaultLoadView, route }: Props) => (
```

Comment:
It was a little bit confusing, that the `SavedSearchPage` component is being used for the `NewSearchPage` and `StreamSearchPage` what about the name `SearchPage`?

Reply:
Having a look at the code I had the following idea. This is a summary of all suggestions. We could:
- rename this component to `SearchPage`
- implement the `SearchPage` for the `NewDashboardPage`. In this case we would use the `SearchPage` for every search page. A side effect would be, that the `ExtendedSearchPage` is only being used inside the `SearchPage`.
- rename `ExtendedSearchPage` to e.g. just `Search` and move it from `pages` to `components/search`

This way the relation between the search pages and the search would be easier to understand, imo.

Reply:
Excellent idea, thanks!